src/HCSR04.h

```diff
@@ -13,8 +13,10 @@ class UltraSonicDistanceSensor {
     /**
      * @param triggerPin  Digital pin that is used for controlling sensor (output).
      * @param echoPin  Digital pin that is used to get information from sensor (input).
+     * @param maxDistanceCm  Maximum distance sensor can measure, defaults to 4m for HC-SR04.
```

Comment:
We could offer more information here -> we could say that it is advisable not to set this bigger than what specification of the sensor says it can do (which is 4m for HC-SR04). And we could say that it can be useful to set it to smaller value if we don't care about distances above that because the measurement will take proportionally less time.